teuthology/provision/__init__.py

```diff
@@ -29,12 +30,17 @@ def reimage(ctx, machine_name, machine_type):
 
     pelagos_types = pelagos.get_types()
     fog_types = fog.get_types()
-    if machine_type in pelagos_types and machine_type in fog_types:
+    maas_types = maas.get_types()
+    if (machine_type in pelagos_types and
```

Comment:
this is no longer the correct test; it should be "only in one of the three _types lists"